### DISCUSSION TITLE
docs/Computer-Science/Algorithm/BFS/

### DISCUSSION THREAD

**Discussion by giscus** [raiders032/youngthree-blog#3] [Q&A]
# docs/Computer-Science/Algorithm/BFS/

1 BFS

https://blog.youngthree.me/docs/Computer-Science/Algorithm/BFS/

**Comment by hanbonghun** [comment]
BFS 연습할만한 기본 문제 추천해주실 수 있나요?

**Reply by raiders032** [reply]
@hanbonghun 
[부대 복귀](https://programmers.co.kr/learn/courses/30/lessons/132266) 이 문제 추천드립니다!
